docs/podman-mount.1.md

```diff
@@ -13,6 +13,8 @@ accessed from the host, and returns its location.
 If you execute the command without any arguments, the tool will list all of the
 currently mounted containers.
 
+Note that the mount command only works with the vfs storage driver in rootless mode.
```

Comment:
suggest:
Note that when running in rootless mode, the mount command only works with the vfs storage driver.

your current wording could lead someone to believe that mount only works for vfs storage in rootless mode and not work with any other driver and not work in root mode.

Reply:
Great suggestions, thanks! I've updated the PR.

Reply:
it is possible to use the mount command also from a user namespace (what `buildah unshare` creates) and there is also a PR for Podman to add `unshare`

Reply:
/hold
Good point. Then let's wait for the unshare PR to get in first. I will update the PR after that.